unattended_installer/install_functions/opendistro/checks.sh

```diff
@@ -308,7 +351,12 @@ function checkPreviousCertificates() {
 
 function checkSpecs() {
 
-    cores=$(cat /proc/cpuinfo | grep -c processor )
+    coresFile=/etc/resolv.conf
+    if [ -f "$coresFile" ]; then
+        cores=$(cat "$coresFile" | grep -c processor )
+    else
+        logger -e "The $coresFile does not exist."
```

Comment:
This is correct, but, we need to expand this to `checkHealth`. Please check what is the `checkHealth` behavior if the `core` variable is empty.

Reply:
Done